types/react-redux/index.d.ts

```diff
@@ -578,4 +579,28 @@ export interface TypedUseSelectorHook<TState> {
  */
 export function useStore<S = any, A extends Action = AnyAction>(): Store<S, A>;
 
+/**
+ * Hook factory, which creates a `useSelector` hook bound to a given context.
+ *
+ * @param Context passed to your `<Provider>`.
+ * @returns A `useSelector` hook bound to the specified context.
+ */
+export function createSelectorHook(context?: Context<ReactReduxContextValue>): typeof useSelector;
```

Comment:
Based on the official docs:
```
const MyContext = React.createContext(null)

// Export your custom hooks if you wish to use them in other files.
export const useStore = createStoreHook(MyContext)
export const useDispatch = createDispatchHook(MyContext)
export const useSelector = createSelectorHook(MyContext)
```

Shouldn't the type for the context argument be `Context<ReactReduxContextValue | null>`? I can create a PR if so.

Reply:
It certainly seems so. The current typings appear to be incompatible with the docs recommendation - at least I was unable to achieve the docs recommendation with the current typings.